dbt_checkpoint/dbt_docs_generate.py

```diff
@@ -6,34 +6,54 @@
 from dbt_checkpoint.utils import (
     add_config_args,
     add_dbt_cmd_args,
+    add_dbt_cmd_model_args,
+    add_filenames_args,
     extend_dbt_project_dir_flag,
     get_config_file,
     get_flags,
+    paths_to_dbt_models,
     run_dbt_cmd,
 )
 
 
-def docs_generate_cmd(
+def prepare_cmd(
```

Comment:
I have changed `docs_generate_cmd`to `prepare_cmd` to be aligned with the rest of the codebase